qualtran/bloqs/factoring/ecc/ec_point.py

```diff
@@ -49,6 +49,8 @@ def __add__(self, other):
         if (other.mod != self.mod) or (other.curve_a != self.curve_a):
             raise ValueError('Use consistent mod and curve')
 
+        if is_symbolic(self.x, self.y, other.x, other.y, self.mod, self.curve_a):
+            return self
```

Comment:
this is misleading. This should be an error probably

Reply:
The reason I did this was to keep tests for the ec_phase_estimate example bloqs. If we change it to raise an error the bloq_autotester fails. I figured if it was symbolic it doesn't really matter what the output is. For now I made it less misleading - if you want me to change to raise an error, I can do that, just have to forego the example bloq tests.

Reply:
where are we adding symbolic EC points in the bloq examples?

Reply:
I see: we manipulate the ECPoint in the decomposition. So is the regression due to the insertion of `int()` wrappers?

Reply:
PTAL at the commit I pushed

Reply:
Yes exactly, we have to use the int wrappers to avoid a mypy typing error and then when there is an int() wrapper on a symbolic value we now also need to have a special case to deal with symbolic values before the wrapper. To remedy that I just added a symbolics check and return a new symbolic value as the result of the addition. Solves all the above problems.

Reply:
LGTM